README.md

```diff
@@ -29,7 +54,11 @@ $ pip install ga4gh-dos-schemas
 ```
 
 This will install both a demonstration server and a Python client that will allow you to
-manage Data Objects in a local server. You can start the demo server using `ga4gh_dos_server`.
+manage Data Objects in a local server.
+
+## Sample Service
+
+You can start the demo server using `ga4gh_dos_server`.
```

Comment:
This should be `ga4gh_drs_server`.

Reply:
OK, I updated the name but the sample service may be slightly broken, need to talk to Natan